lib/logstash/filters/split.rb

```diff
@@ -57,20 +57,18 @@ class LogStash::Filters::Split < LogStash::Filters::Base
   # The field within the new event which the value is split into.
   # If not set, the target field defaults to split field name.
   config :target, :validate => :string
-
+  
```

Comment:
I suggest removing this whitespace changes as they don't contribute to the purpose of the PR